apiserver/facades/agent/uniter/service.go

```diff
@@ -125,6 +126,9 @@ type ApplicationService interface {
 	// If no application is found, an error satisfying
 	// [applicationerrors.ApplicationNotFound] is returned.
 	GetApplicationConfig(ctx context.Context, uuid coreapplication.ID) (coreconfig.ConfigAttributes, error)
+
+	// GetUnitRefreshAttributes returns the refresh attributes for the unit.
+	GetUnitRefreshAttributes(ctx context.Context, unitName coreunit.Name) (domainapplication.UnitAttributes, error)
```

Comment:
todo: We agree that `Refresh` is a horrid facade name. Can we not add more of it? `GetUnitAttributes` is a better descriptor. 

same for the `getRefresh` method.

Reply:
> GetUnitAttributes is a better descriptor.

This implies that there is something about the unit that has attributes, which isn't the case. I could potentially just call `GetUnitRefreshInfo` - might be a better name for the service.